Added form-control for email element in webform.

tkcent opened the pull request:
Fixes https://github.com/backdrop-contrib/bootstrap_lite/issues/44.

The webform email element needs an additional theme call to add the form-control class.

tkcent committed:
Added form-control for email element in webform.

The webform email element needs an additional theme call to add the form-control class.

bugfolder commented on template.php:
```diff
@@ -300,6 +300,22 @@ function bootstrap_lite_email($variables) {
+ *
+ * @ingroup themeable
+ */
+function bootstrap5_lite_webform_email($variables) {
```
Should be `bootstrap_lite_webform_email`, not `bootstrap5_lite_webform_email`.

tkcent committed:
Fixed function name for bootstrap_lite_webform_email

Updated function in the wrong repository (there is also a bootstrap5_lite).  This theming function is still needed for webform.

tkcent commented:
Whoops, I didn't catch there were two similarly named themes!   I just verified the patch is needed here as well though.  I fixed function name in my repository.  Do I close this issue and open a new pull request on my repository?
